index.js

```diff
@@ -53,6 +53,7 @@ module.exports = {
     'mocha/no-pending-tests': 'error',
     'mocha/no-skipped-tests': 'warn',
     'mocha/valid-test-description': 'error',
+    'no-template-curly-in-string': 'off',
```

Comment:
Bunsen does not need its own ESLint plugin as you suggested with `eslint-plugin-<bunsen>` - that is not "how it works" or how we are using it.  As @juwara0 referenced there is an _.eslintrc_ file in Bunsen - https://github.com/ciena-frost/ember-frost-bunsen/blob/master/.eslintrc - where Bunsen-specific modifications should reside.

You are correct that this repo serves as a place to hold our of our ESLint rules but before making this change in this PR have you determined the impact of changing this rule?  Does any other repo use or require syntax like this?